reference/activity_manager/project_completeness.rst

```diff
@@ -0,0 +1,110 @@
+What is the project completeness?
+=================================
+
+Overview
+________
+
+In this reference guide you will learn what the project completeness refers to and how we manage it in Akeneo PIM. It
+will explain our technical and structure choices.
+
+Goal
+____
+
+The Activity Manager feature is about creating project from a products selection that give vision on the enrichment
+progress to contributors (users in charge of enrichment). Each project gives to the contributor its own completeness on
+the products selection, taking in account its permissions to edit or not products, attribute groups, locales, scopes.
+
+For example, if Bob can only edit the "Technical" attribute group of "High-tech" category, he will see the project 100%
+complete if all attributes in attribute group "Technical" of "High-tech" products in the project are filled, even if
+other attribute groups are empty and required by the family. However, Julia, the owner of the project, will see that it
+is not complete if some attributes are not filled in others attribute groups. The contributor see the completeness of
+the project in terms of his own permissions, whereas the owner see the completeness of all the project.
+
+As every user will have his very own completeness depending on his permissions, we need to walk through every products
+to compute completeness for every user... As you can guess, it's a really huge task that cannot be done synchronously.
+
+We need to go deeper
```

Comment:
I'd love to see some of the information provided in this paragraph in an ADR! This is mainly about implementations issues you found and the motivations that drove you to choose another solution. (Rather than in the documentation but that's fine it is ;) )